dataframe/dataframe.go

```diff
@@ -656,6 +657,101 @@ func WithTypes(coltypes map[string]series.Type) LoadOption {
 	}
 }
 
+// Load creates a new DataFrame from arbitrary []structs
+func Load(i interface{}) DataFrame {
```

Comment:
The name Load is not representative of the values this function consumes. It should be changed to LoadStructs instead.

In addition to this, this function should support the configuration of LoadOptions.